content/tutorials/basic-movement.md

```diff
@@ -2,264 +2,164 @@
 title: Basic Movement
 ---
 
-The aim of this tutorial is to get a motor turning with your kit.
-To complete this tutorial, you'll need the following:
+The aim of this tutorial is to get a motor turning with your kit. To complete this tutorial, you'll need the following:
 
-* The [power board](/docs/kit/power_board)
+* The [power board](/kit/power-board)
+* A [motor board](/kit/motor-board)
+* The [Pi](/kit/pi)
 * The battery (charged, of course)
-* The cable to connect the battery to the power board
-* A [motor board](/docs/kit/motor_board)
 * 2 large (7.5mm) green _CamCon_ connectors to plug the power board and motor board together
 * 2 lengths of a suitable gauge of wire for powering the motor board from the power board (one should be black/grey)
```

Comment:
Although this line wasn't touched by your PR, I think changing "suitable gauge of wire" to "suitably thick wire" would make this line clearer to those who aren't familiar with the concept of "wire gauge".

Reply:
Any changes to the file are relevant, not just my changes 👍 But yes good change idea!